subprojects/android-test/test-report/src/main/kotlin/com/avito/android/test/report/troubleshooting/dump/MainLooperMessagesLogDumperImpl.kt

```diff
@@ -0,0 +1,26 @@
+package com.avito.android.test.report.troubleshooting.dump
+
+import android.os.Looper
+import android.util.Printer
+
+class MainLooperMessagesLogDumperImpl : Printer, MainLooperMessagesLogDumper {
+
+    private val logs = StringBuilder()
+
+    override fun println(log: String) {
+        logs.appendLine(log)
+    }
```

Comment:
We could reduce the size of the dump by removing duplicated prefixes in messages:

```
>>>>> Dispatching to Handler (android.os.Handler) {3fbc37c} t.b@15aa3405: 0
<<<<< Finished to Handler (android.os.Handler) {3fbc37c} t.b@15aa3405
>>>>> Dispatching to Handler (android.os.Handler) {3fbc37c} t.b@162fdc5a: 0
<<<<< Finished to Handler (android.os.Handler) {3fbc37c} t.b@162fdc5a
>>>>> Dispatching to Handler (android.os.Handler) {3fbc37c} t.b@f206e8b: 0
<<<<< Finished to Handler (android.os.Handler) {3fbc37c} t.b@f206e8b
>>>>> Dispatching to Handler (android.os.Handler) {3fbc37c} t.b@1cff9b68: 0
```

I've checked on samples. These "Dispatching ..." prefixes are about 15% of the content.

Reply:
Good idea. Could you write a solution?

Reply:
Yes. Will do a PR

Reply:
#887